store/pruning.go

```diff
@@ -0,0 +1,29 @@
+package store
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// default pruning strategies
+var (
+	// PruneEverything means all saved states will be deleted, storing only the current state
+	pruneEverything = sdk.NewPruningOptions(0, 0)
+	// PruneNothing means all historic states will be saved, nothing will be deleted
+	pruneNothing = sdk.NewPruningOptions(0, 1)
+	// PruneSyncable means only those states not needed for state syncing will be deleted (keeps last 100 + every 10000th)
+	pruneSyncable = sdk.NewPruningOptions(100, 10000)
+)
+
+func NewPruningOptions(strategy string) (opt PruningOptions) {
+	switch strategy {
+	case "nothing":
```

Comment:
These should be constants defined in this file and used everywhere `NewPruningOptions` is called imho.

Reply:
We still need this method to parse viper flag value. To be able to change default pruning options, **baseapp** should take not just strategy, but options itself. So we can't just pass **strategy** to ** baseapp**. Maybe I miss something?
Btw, I open **PruneEverything**, **PruneNothing**, **PruneSyncable** and change tests to use them, instead of this method.

Reply:
I think this is OK for now.